incidents_mttr_example.py

```diff
@@ -7,45 +7,56 @@
 # Version:      1.0
 #
 
-import json
 import argparse
 from datetime import date, datetime, timedelta
 import csv
 import demisto
 
+
 def format_dt(dt):
     return dt.strftime('%Y-%m-%dT%H:%M:%S')
 
+
 def parse_dt(dt):
     return datetime.strptime(dt[:19], '%Y-%m-%dT%H:%M:%S')
 
+
 def p(what):
     if verbose:
         print(what)
 
+
 def options_handler():
-    parser = argparse.ArgumentParser(description='Incident MTTR statistics for a time period')
-    parser.add_argument('-k', '--key', help='The API key to access the server', required=True)
-    parser.add_argument('-s', '--server', help='The server URL to connect to', required=True)
+    parser = argparse.ArgumentParser(
+        description='Incident MTTR statistics for a time period')
+    parser.add_argument(
+        '-k', '--key', help='The API key to access the server', required=True)
+    parser.add_argument(
+        '-s', '--server', help='The server URL to connect to', required=True)
     monthAgo = date.today() - timedelta(days=30)
-    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is closed incidents in last month', default='status:=2 and closed:>%s' % format_dt(monthAgo))
-    parser.add_argument('-g', '--group', help='The field to group by to calculate mttr, default is owner', default='owner')
-    parser.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True, help="no extra prints")
-    parser.add_argument('-o', '--output', help='The output CSV file', default='mttr.csv')
+    parser.add_argument('-f', '--filter', help='The filter query to chose the alerts, default is closed incidents in last month',
+                        default='status:=2 and closed:>%s' % format_dt(monthAgo))
+    parser.add_argument(
+        '-g', '--group', help='The field to group by to calculate mttr, default is owner', default='owner')
+    parser.add_argument('-q', '--quiet', action='store_false',
+                        dest='verbose', default=True, help="no extra prints")
+    parser.add_argument(
+        '-o', '--output', help='The output CSV file', default='mttr.csv')
```

Comment:
Why the inconsistent indentation between args?

Reply:
fix strange indent (49e8339).